### DISCUSSION TITLE
App Icon image

### DISCUSSION THREAD

**Discussion by GalaxyM105** [omnivore-app/omnivore#3968] [Ideas]
Hey Omnivore devs,

I am on Google Pixel 8 pro and I recently noticed that the app icon has a new image on it with the "O" with a bite taken out of it at the bottom right corner. 🙂 I like that idea, I think maybe incorporating an animal like a bear (which is an Omnivore), would look pretty cool. 🙂 Maybe reading some papers or something.

I didn't see a discussion post for the app icon already so I thought I would go ahead and make one. 🙂

**Comment by babetoduarte** [comment]
Love the idea for the icon too, but I wish it was an adaptive icon 😅 hopefully in an upcoming update 🤞🏻

**Comment by pusherbox** [comment]
IMHO the old icon was waaaay better... So why don't give the chance to choose it?
